src/metatensor/models/utils/evaluate_model.py

```diff
@@ -43,9 +48,12 @@ def evaluate_model(
 
     :returns: The predictions of the model for the requested targets.
     """
+    if is_exported(model):
+        outputs_capabilities = model.capabilities().outputs
+    else:
+        outputs_capabilities = model.capabilities.outputs
 
     # Assert that all targets are within the model's capabilities:
-    outputs_capabilities = _get_capabilities(model).outputs
     if not set(targets.keys()).issubset(outputs_capabilities.keys()):
```

Comment:
Can we keep `_get_capabilities` here? It's going to get larger with distributed training